frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -795,34 +795,15 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
     }
   }, [filterSelectionsStr, internalSearch, isPreProcessed, page, perPage, setRequestView, sort])
 
-  const { tableItems, totalCount } = useMemo<{
+  const { tableItems, totalCount, allTableItems } = useMemo<{
     tableItems: ITableItem<T>[]
     totalCount: number
+    allTableItems: ITableItem<T>[]
   }>(() => {
     /* istanbul ignore if */
-    if (!items) return { tableItems: [], totalCount: 0 }
-    let filteredItems: T[] = items
+    if (!items) return { tableItems: [], totalCount: 0, allTableItems: [] }
 
-    // if using a result view from backend, the items have already been filtered
-    if (!isPreProcessed) {
-      if (filters.length && Object.keys(filterSelections).length) {
-        filteredItems = applyFilters(items, filterSelections, filters)
-      }
-
-      // advanced filtering
-      const advancedFilterSelections: CurrentFilters<AdvancedFilterSelection> = {}
-      activeAdvancedFilters.forEach(({ columnId, value, operator }) => {
-        if (columnId && value && operator) {
-          advancedFilterSelections[columnId] = { operator, value }
-        }
-      })
-
-      if (advancedFilters.length && Object.keys(advancedFilterSelections).length) {
-        filteredItems = applyFilters(filteredItems, advancedFilterSelections, advancedFilters)
-      }
-    }
```

Comment:
The diff picked up primarily on the migration of this code to lines 825+
The code building the tableItem object array now precedes this code.